sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -1054,6 +1054,15 @@ trait ShowCreateTableCommandBase {
   private def showViewText(metadata: CatalogTable, builder: StringBuilder): Unit = {
     builder ++= metadata.viewText.mkString("AS ", "", "\n")
   }
+
+  def quoted(identifier: TableIdentifier): String = {
```

Comment:
is it simply `identifier.asMultiPartIdentifier.quoted`?

Reply:
`TableIdentifier` could not `asMultiPartIdentifier`, `Identifier` can `asMultiPartIdentifier`